qcore/geo.py

```diff
@@ -465,6 +465,58 @@ def avg_wbearing(angles):
     return degrees(atan(x / y) + q_diff)
 
 
+def build_corners(origin, rot, xlen, ylen):
+    # wanted xlen, ylen is at corners
```

Comment:
Docstring? Specifically looking for what's the format of the return type.

Reply:
Done.